bubble_sort.rb

```diff
@@ -0,0 +1,36 @@
+def bubble_sort(arr)
+    n=arr.length
+    loop do
+        swapped=false
```

Comment:
make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

`swapped=false`     #bad
`swapped = false`   #good

correct this and go head and merge to master.